docker-compose.e2e-basic.yml

```diff
@@ -176,7 +186,81 @@ services:
       --openvpn.port=3000
       openvpn,noop,wireguard
 
-  myst-consumer:
+  myst-consumer-noop:
```

Comment:
Lets have one consumer and create sessions from it. Doing consequent connects/disconnect showed bunch of bugs for us.
How to keep testing that?

Reply:
maybe additional connect after 1st session?

Reply:
Actually connects should happen in same container, to catch race bugs for us.
But we can run those connect tests in parallel

Reply:
I leave this an exercise to the reader. Need a break from e2e for now.